src/netaddress.h

```diff
@@ -16,6 +16,9 @@
 #include <string>
 #include <vector>
 
+static const std::string ONION{".onion"};
```

Comment:
56b156dc161b766e4aaae1ca07c719a3c599c4cb

```suggestion
static const std::string ONION_DOMAIN{".onion"};
```

Reply:
done